pkg/virt-controller/watch/vm/vm.go

```diff
@@ -808,6 +808,25 @@ func (c *Controller) handleVolumeRequests(vm *virtv1.VirtualMachine, vmi *virtv1
 	return nil
 }
 
+func (c *Controller) handleDeclarativeVolumeHotplug(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
General comment, you are adding a lot of new function for storage, can we please create and move them into storage.go . We do something similar for the vmi

Reply:
good idea